### DISCUSSION TITLE
mddatasetbuilder does not work properly.

### DISCUSSION THREAD

**Discussion by nnyyxx-bit** [tongzhugroup/mddatasetbuilder#183] [General]
After installing mddatasetbuilder by pip install, however, I get the following error at runtime, how do I fix this.
![屏幕截图 2023-09-08 203335](https://github.com/tongzhugroup/mddatasetbuilder/assets/144452397/172a0d69-abaa-49e9-8bb4-bc3076d08577)
![屏幕截图 2023-09-08 203415](https://github.com/tongzhugroup/mddatasetbuilder/assets/144452397/599fe7cf-d1ad-402c-8c2a-db056467cc3e)

**Comment by njzjz** [comment]
Thanks for reporting. The error has been fixed in #184.

**Reply by njzjz** [reply]
```sh
pip install -U mddatasetbuilder
```

If the gaussianrunner package is also broken,
```sh
pip install -U gaussianrunner
```

**Reply by nnyyxx-bit** [reply]
Thank you for your reply. I have solved this problem according to your method. But a new problem has arisen, and I did not find 03.data in this path. How should I find it.
![1](https://github.com/tongzhugroup/mddatasetbuilder/assets/144452397/215bd706-e884-413e-8e3e-a2360a8a020d)
![2](https://github.com/tongzhugroup/mddatasetbuilder/assets/144452397/0d1d01b8-eb4a-4c34-abda-5f1dc136a7b0)

**Reply by njzjz** [reply]
I see that the `02.fp` directory is generated, so Gaussian calculation seems to have been performed.

You can check the log files:

1. If these calculations failed to be performed or failed to coverage, check what happened with Gaussian;
2. If the force is not outputted, check if you have `force` in the keyword;
3. If the input coordinate is not outputted, check if you have `Geom=PrintInputOrient` in the keyword.

**Comment by nnyyxx-bit** [comment]
1123875178
***@***.***

Thank you for your answer, but I found that the reason for my error was the use of "mn15/6-31g**", which was also set according to the DP-GUI recommendation, and instead of "b3lyp/6-31g**", the initial data was successfully generated. In addition, would you like to ask MDDatasetbuilder sampling structure is a single molecule? That's not the structure I sampled. Looking forward to your reply.

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tongzhugroup/mddatasetbuilder"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月14日(星期四) 凌晨3:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [tongzhugroup/mddatasetbuilder] mddatasetbuilder does not work properly. (Discussion #183)





 
I see that the 02.fp directory is generated, so Gaussian calculation seems to have been performed.
 
You can check the log files:
  
If these calculations failed to be performed or failed to coverage, check what happened with Gaussian;
 
If the force is not outputted, check if you have force in the keyword;
 
If the input coordinate is not outputted, check if you have Geom=PrintInputOrient in the keyword.

**Reply by njzjz** [reply]
MDDatasetbuilder generates a cluster where a central atom and the molecules within the cutoff radius of the central atom are included. Please refer to Nature Communications, 11, 5713 (2020) for details.
